nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/test/java/org/apache/nifi/controller/TestStandardFlowFileQueue.java

```diff
@@ -71,12 +72,12 @@ public class TestStandardFlowFileQueue {
 
     private List<ProvenanceEventRecord> provRecords = new ArrayList<>();
 
-    @BeforeClass
+    @BeforeAll
     public static void setupLogging() {
         System.setProperty("org.slf4j.simpleLogger.log.org.apache.nifi", "INFO");
     }
```

Comment:
This method should be removed since additional logging should not be enabled as a general rule.